### ISSUE TITLE
Rel8 function

### ISSUE THREAD

**Issue by bisoldi** [dbpedia-spotlight/dbpedia-spotlight#444]
Hi all,

I'm wondering if the rel8 function is working completely?  I've been unable to hit the endpoint while deploying both the jar and the docker image.  I then noticed that to build it, additional steps need to be taken to add the Lucene index, so I wasn't sure if it's operational in the deployed jar / docker.

Thanks!

**Comment by sandroacoelho** [comment]
Hi @bisoldi 
We are moving our model's extractor to Jenkins. Since today, I need five business day more to provide you a new Lucene backend. I tell you when are available.

Best,

**Comment by sandroacoelho** [comment]
Hi @bisoldi ,

As I promised, our Lucene backend is available under http://downloads.dbpedia-spotlight.org/2016-04/en/lucene/. 

Have you ever considered using DBpedia Lookup (https://github.com/dbpedia/lookup) instead? 

@pablomendes: Do you know if there is some feature in Rel8 that is not available there?

Best,

**Comment by bisoldi** [comment]
Hi @sandroacoelho ,

My hope in using dbpedia-spotlight's Rel8 function was to be able to give it a term and simply get back related terms.  So, for example, if I gave it "artificial intelligence", I was looking for a list of terms back such as "machine learning", "neural network", etc.  Or if I gave it "cybersecurity", I would get back terms such as "sniffing", "spoofing", "phishing", "0-day exploits", etc.

Is that something the DBpedia Lookup can do?  I looked at the keyword search and while it looks like it provides terms that are domain-related, they don't really fall under the term itself.  For example, querying "berlin" returned "West Berlin", "Berlin Wall", "Axis Powers" and "Kinshasa".  I'm not sure why Kinshasa is returned, but the other three are certainly related, but not necessarily the type of relationship I was hoping for.

Your thoughts and advice are welcome!!

**Comment by sandroacoelho** [comment]
Yeah. You are right. DBpedia Lookup will not fit in what you are looking for. 

Give me chance to try to put REL8 working again. Then we can decide to move features to Lookup or keep it in Spotlight.

Best,
